src/wallet/rpcwallet.cpp

```diff
@@ -1650,7 +1655,7 @@ UniValue listtransactions(const UniValue& params, bool fHelp)
     int nFrom = 0;
     if (params.size() > 2)
         nFrom = params[2].get_int();
-    isminefilter filter = ISMINE_SPENDABLE;
+    isminefilter filter = ISMINE_LEGACY_SPENDABLE;
```

Comment:
Hm... do we actually provide any other mechanism yet to obtain a list of transactions for an account? I think that with this change, it means that #5032 / #1438 (in general, having a single source of truth for wallet transaction history) will need to also be completed in v4.6.0. If they're not, this should be reverted.